### PR TITLE
Add Enterprise account type

### PR DESCRIPTION
**Pull request by msya**
Added enterprise account type enum value to user object.

### CODE REVIEW

File: vimeo-networking/src/main/java/com/vimeo/networking/model/User.java [hunk 1]
```diff
@@ -69,7 +70,8 @@
         LIVE_PRO,
         LIVE_PREMIUM,
         PRO_UNLIMITED,
-        PRODUCER
+        PRODUCER,
+        ENTERPRISE
```
Comment (anthonycr) [inline vimeo-networking/src/main/java/com/vimeo/networking/model/User.java:74]: Enterprise should also be added to `networking2.enums.AccountType`.

**Review by anthonycr — CHANGES_REQUESTED** [review]
Networking V2 should also be updated.

**Comment by msya** [comment]
@anthonycr Back to you.

**Review by anthonycr — APPROVED** [review]
LGTM

**Pull request merged**